### ISSUE TITLE
Auto Tab complete benchmarks

### ISSUE THREAD

**Issue by jamescho72** [redhat-developer/vscode-paver#134]
Lavanya please reach out to Max and bring in his tab complete code generation tests.

Work with rest of team to automate the testing with our existing testcases.

**Comment by owtaylor** [comment]
I don't think this is relevant in this form any more - we're now working within the BigCodeBench framework and that has code completion tests we could pick out / extend.
